tests/test_events.py

```diff
@@ -360,55 +360,6 @@ class TestVideo(asynctest.TestCase):
-    props = {
-        "index": 0,
-        "codec_name": "h264",
-        "codec_long_name": "H.264 / AVC / MPEG-4 AVC / MPEG-4 part 10",
-        "profile": "Main",
-        "codec_type": "video",
-        "codec_time_base": "1/10",
-        "codec_tag_string": "[0][0][0][0]",
-        "codec_tag": "0x0000",
-        "width": 32,
-        "height": 32,
-        "coded_width": 32,
-        "coded_height": 32,
-        "has_b_frames": 2,
-        "sample_aspect_ratio": "16:9",
-        "display_aspect_ratio": "16:9",
-        "pix_fmt": "yuv420p",
-        "level": 31,
-        "color_range": "tv",
-        "color_space": "smpte170m",
-        "color_transfer": "smpte170m",
-        "color_primaries": "smpte170m",
-        "chroma_location": "left",
-        "field_order": "progressive",
-        "refs": 1,
-        "is_avc": "true",
-        "nal_length_size": "4",
-        "r_frame_rate": "5/1",
-        "avg_frame_rate": "5/1",
-        "time_base": "1/1000",
-        "start_pts": 0,
-        "start_time": "0.000000",
-        "bits_per_raw_sample": "8",
-        "disposition": {
-            "default": 1,
-            "dub": 0,
-            "original": 0,
-            "comment": 0,
-            "lyrics": 0,
-            "karaoke": 0,
-            "forced": 0,
-            "hearing_impaired": 0,
-            "visual_impaired": 0,
-            "clean_effects": 0,
-            "attached_pic": 0,
-            "timed_thumbnails": 0,
-        },
-        "tags": {"DURATION": "00:00:01.000000000"},
-    }
```

Comment:
The properties are changed because of the new version of ffmpeg (4.4.2 instead of 4.2.7). Because of this the properties output is slightly different.
I found out the difference, if we decide to retain the test:

```
diff --git a/tests/test_events.py b/tests/test_events.py
index db0e65f..72c1683 100644
--- a/tests/test_events.py
+++ b/tests/test_events.py
@@ -366,13 +366,13 @@ class TestVideo(asynctest.TestCase):
         "codec_long_name": "H.264 / AVC / MPEG-4 AVC / MPEG-4 part 10",
         "profile": "Main",
         "codec_type": "video",
-        "codec_time_base": "1/10",
         "codec_tag_string": "[0][0][0][0]",
         "codec_tag": "0x0000",
         "width": 32,
         "height": 32,
         "coded_width": 32,
         "coded_height": 32,
+        "closed_captions": 0,
         "has_b_frames": 2,
         "sample_aspect_ratio": "16:9",
         "display_aspect_ratio": "16:9",

```